sdk/iot/Azure.Iot.Hub.Service/src/Customized/Models/DigitalTwinData.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Iot.Hub.Service.Models
+{
+    /// <summary>
+    /// Digital twin data.
+    /// </summary>
+    [CodeGenModel("Twin")]
+    internal partial class DigitalTwinData
```

Comment:
This is the device/ module twin, not the digital twin (pnp). I don't have a good suggestion for the name though :(

Reply:
Does the summary say that this is digital twin data?

Reply:
There was no summary in our updated swagger. Have you added it already?

Reply:
No, the twin APIs PR isn't out yet.

Reply:
DeviceTwin maybe?

Reply:
The thing here is that this could be device twin or module twin

Reply:
Then what to do about moduleTwin? :P

Reply:
Twin may be the best name here, still. We could talk to the Azure.Core folks about allowing such a short name

Reply:
I'll call it TwinData for now to unblock ourselves and we can go from there.